pkgs/development/compilers/gcc/8/default.nix

```diff
@@ -65,12 +65,13 @@ let majorVersion = "8";
     /* Cross-gcc settings (build == host != target) */
     crossMingw = targetPlatform != hostPlatform && targetPlatform.libc == "msvcrt";
     stageNameAddon = if crossStageStatic then "-stage-static" else "-stage-final";
-    crossNameAddon = if targetPlatform != hostPlatform then "-${targetPlatform.config}" + stageNameAddon else "";
+    crossNameAddon = if targetPlatform != hostPlatform then "${targetPlatform.config}${stageNameAddon}-" else "";
```

Comment:
```suggestion
    crossNameAddon = opionalString (targetPlatform != hostPlatform) "${targetPlatform.config}-${stageNameAddon}-";
```

Reply:
OK, I think I have it fixed now. Do you know how I can test it?